buildme

```diff
@@ -1,53 +1,31 @@
 #!/bin/bash
-BUILDTYPE=Release
-ARCH=$(uname -m)
-ARM64=OFF
-CMAKE_TOOLCHAIN_FILE=../../../makefiles/cmake/toolchains/arm-linux-gnueabihf.cmake
 
-if [ "$1" = "--debug" ]; then
-	BUILDTYPE=Debug
-	shift
-fi
+CMAKE_BUILD_TYPE=Release
+CMAKE_TOOLCHAIN_FILE=/home/andreas/Code/Android/SDK/ndk/21.3.6528147/build/cmake/android.toolchain.cmake
 
-if [ "$1" = "--aarch64" ]; then
-	ARM64=ON
-	CMAKE_TOOLCHAIN_FILE=../../../makefiles/cmake/toolchains/aarch64-linux-gnu.cmake
-	shift
-fi
+ANDROID_ABI=armeabi-v7a
+ANDROID_TOOLCHAIN=clang
+ANDROID_ARM_NEON=ON
+ANDROID_NATIVE_API_LEVEL=24
```

Comment:
Does this mean we have to make different prebuilds for different Android versions ? Android 7.1 has API 25 and Android 9 uses API 28 why do we use 24 here ?

Reply:
I adopt this parameter from our old build script, too. I found this:

Specifies the minimum API level supported by the application or library. This value corresponds to the application's minSdkVersion. [See also](https://developer.android.com/ndk/guides/cmake#android_platform).

So I guess a smaller value shouldn't make it bader.